homeassistant/components/lg_thinq/config_flow.py

```diff
@@ -83,8 +83,8 @@ async def async_step_user(
 
             try:
                 return await self._validate_and_create_entry(access_token, country_code)
-            except ThinQAPIException:
-                errors["base"] = "token_unauthorized"
+            except ThinQAPIException as exc:
+                errors["base"] = exc.error_name.lower()
```

Comment:
This isn't good practice. This couples our translations and error directly with properties of an external library.

This should be de-coupled.

Reply:
@frenck 
Added THINQ_ERRORS mapping for translation_key and default is "token_unauthorized".